packages/@ember/-internals/glimmer/lib/helpers/gt.ts

```diff
@@ -0,0 +1,41 @@
+import { Arguments, CapturedArguments, VM } from '@glimmer/runtime';
+import { InternalHelperReference } from '../utils/references';
+
+/**
+@module ember
+*/
+
+/**
+  Compares the two given values with >
+
+  Example:
+
+  ```handlebars
+  {{gt age 17}}
+
+  {{! be true if `age` is > 17}}
+  ```
+
+  @public
+  @method gt
+  @for Ember.Templates.helpers
+  @since 2.7.0
+*/
+function gt({ positional: { references } }: CapturedArguments) {
+  let left = references[0].value();
+  if (left === undefined || left === null) {
+    return false;
+  }
+  let right = references[1].value();
+  if (right === undefined) {
```

Comment:
Does this satisfy a specific edge case? I would have imagined that the following would have been enough:


```js
return left > right;
```

Reply:
I tried, but typescript doesn't like comparing things with null.